app/controllers/v1/sessions/report_templates_controller.rb

```diff
@@ -67,6 +69,16 @@ def remove_logo
     render status: :ok
   end
 
+  def remove_cover_letter_custom_logo
+    authorize! :remove_cover_letter_custom_logo, report_template
+
+    return head :forbidden unless @session.ability_to_update_for?(current_v1_user)
+
+    report_template.cover_letter_custom_logo.purge
+
+    render status: :ok
+  end
```

Comment:
Do you think it looks exactly the same as the method above? Do you have any idea how to use the DRY principle?